odxtools/multiplexer.py

```diff
@@ -120,14 +135,20 @@ def from_et(et_element, doc_frags: List[OdxDocFragment]) -> "MultiplexerSwitchKe
             dop_ref=dop_ref,
         )
 
-    def _resolve_references(self, odxlinks: OdxLinkDatabase) -> None:
+    def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
+        return {}
```

Comment:
Why have the function if not going to implement it?

Reply:
uniform API: you do not need to handle any sub-objects differently, regardless of whether they have an ID or not, need to resolve ODXLINKs or not or need to resolve SNREFs or not...